web/cypress/e2e/route/create-route-both-use-uri-uris.cy.js

```diff
@@ -72,7 +72,7 @@ context('Create Route Both use uri and uris', () => {
     cy.contains('Route').click();
     cy.get(selector.empty).should('be.visible');
     cy.contains('Create').click();
-    cy.contains('Next').click().click();
```

Comment:
The two clicks here are to ensure that the rendering of the page is finished, and this is a point that can be optimized. 
Please try to restore it first